content/manuals/desktop/release-notes.md

```diff
@@ -30,6 +30,41 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 For more frequently asked questions, see the [FAQs](/manuals/desktop/troubleshoot-and-support/faqs/releases.md).
 
+## 4.49.0
+
+{{< release-date date="2025-10-23" >}}
+
+{{< desktop-install-v2 all=true win_arm_release="Early Access" version="4.49.0" build_path="/TODO/" >}}
+
+> [!IMPORTANT]
+>
+> Support for Windows 10 and 11 22H2 (19045) has ended. Installing Docker Desktop will require Windows 11 23H2 in the next release.
```

Comment:
Should we link to the system requirements section?

Reply:
It's just a repeat of this information, so I don't feel we need to